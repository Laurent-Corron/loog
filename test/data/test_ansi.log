2019-12-22 11:52:20,441 7214 [1;32m[1;49mINFO[0m ? odoo: Odoo version 12.0
2019-12-22 11:52:20,626 7214 [1;33m[1;49mWARNING[0m ? odoo.addons.base.models.res_currency: The num2words python library is not installed, amount-to-text features won't be fully available.
Exception in thread odoo.service.httpd:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
