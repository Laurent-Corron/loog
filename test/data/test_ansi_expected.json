[
  {
    "asctime": "2019-12-22 11:52:20,441",
    "pid": "7214",
    "levelname": "INFO",
    "dbname": "?",
    "logger": "odoo",
    "message": "Odoo version 12.0"
  },
  {
    "asctime": "2019-12-22 11:52:20,626",
    "pid": "7214",
    "levelname": "WARNING",
    "dbname": "?",
    "logger": "odoo.addons.base.models.res_currency",
    "message": "The num2words python library is not installed, amount-to-text features won't be fully available.\nException in thread odoo.service.httpd:\nTraceback (most recent call last):\nFile \"/usr/lib/python3.7/threading.py\", line 926, in _bootstrap_inner\nself.run()"
  }
]
